app/models/controlled_vocabulary.rb

```diff
@@ -98,6 +102,40 @@ def all(_scope = nil)
     end
   end
 
+  class GeoTerm < Valkyrie::Resource
```

Comment:
Why a special GeoTerm? Concerned about the returned thing from a ControlledVocabulary having different methods.

Reply:
I think that'll work. I'll just redefine `code` as `definition`.